### PR TITLE
Fixes issue with initializing Woodwork Series with LatLong values

### LINKED ISSUE #1298: Initialize Series with tuple of nans fails
Initializing a series with a tuple of nans fails.

#### Code Sample

```python
ww.init_series(pd.Series([(1.0, 2.0), (float('nan'), float('nan'))]), logical_type='LatLong')
```

Returns this error
```
ValueError: Cannot initialize Woodwork. Series does not contain properly formatted LatLong data. Try reformatting before initializing or use the woodwork.init_series function to initialize.
```

### PR DESCRIPTION
**Pull request by dvreed77**
- Fixes issue with initializing Woodwork Series with LatLong values
- Closes #1298

### CODE REVIEW

File: docs/source/release_notes.rst
Comment (thehomebrewnerd) [inline docs/source/release_notes.rst:17]: Add your username to the list of contributors below.

File: woodwork/tests/utils/test_accessor_utils.py
Comment (thehomebrewnerd) [inline woodwork/tests/utils/test_accessor_utils.py:104]: Is there a reason for testing this with only the pandas version of the fixture and not testing with Dask & Koalas?

File: woodwork/utils.py
Comment (thehomebrewnerd) [inline woodwork/utils.py:218]: The docstring for this needs to be updated now that we can also have a single `nan` value. Probably should also review all the other latlong related docstrings to make sure they are consistent with the new behavior as well.

File: woodwork/tests/utils/test_utils.py
Comment (thehomebrewnerd) [inline woodwork/tests/utils/test_utils.py:285]: Do we need to test any more input types here - like these for example:
```
_reformat_to_latlong(np.nan)
_reformat_to_latlong(pd.NA)
_reformat_to_latlong((np.nan, np.nan))
_reformat_to_latlong((pd.NA, pd.NA))
_reformat_to_latlong((None, None))
```

File: woodwork/tests/utils/test_utils.py
Comment (thehomebrewnerd) [inline woodwork/tests/utils/test_utils.py:357]: Since we are generally supporting nullable columns types, may be we should test with some `pd.NA` inputs as well and make sure those return False?

File: docs/source/release_notes.rst
Comment (gsheni) [reply docs/source/release_notes.rst:17]: done

File: woodwork/tests/utils/test_utils.py
Comment (rwedge) [inline woodwork/tests/utils/test_utils.py:295]: if the only difference between `test_reformat_to_latlong` and `test_reformat_to_latlong_koalas`, could parametrize `is_koalas` True/False and cast from list to tuple or vice versa if expected needs to be a list and is a tuple

File: woodwork/exceptions.py
Comment (rwedge) [inline woodwork/exceptions.py:86]: This error message does not mention a NaN value is ok.

File: woodwork/utils.py
Comment (rwedge) [inline woodwork/utils.py:301]: can this be hit?

File: woodwork/exceptions.py
Comment (thehomebrewnerd) [inline woodwork/exceptions.py:83]: Do we want/need separate error classes for each of these, or should we use a single class and customize the error based on the specific failure? Also, is this distinct from the existing `TypeValidationError` or could we use that instead? If we don't use `TypeValidationError` should these be a subclass of that?

I'm not sure what the right answer is to any of those questions, just raising it for discussion.

**Review by thehomebrewnerd — APPROVED** [review]
This looks good to me, but might not hurt to get a second opinion before you merge this in.

**Review by tamargrey — APPROVED** [review]
lgtm!

**Pull request merged**
